core/src/price_estimation/clients/dexag/api.rs

```diff
@@ -89,6 +88,10 @@ impl DexagApi for DexagHttpApi {
         }
         .boxed()
     }
+
+    fn stable_coin_symbol() -> String {
```

Comment:
ubertinynanonit (feel free to ignore if you otherwise wouldn't touch this): could return &str and caller can decide to own it.